pkg/key/key.go

```diff
@@ -0,0 +1,9 @@
+package key
+
+const (
+	CalicoVersionConstraint                       = ">= 3.10.0 < 3.15.0"
```

Comment:
Azure 12.0.0 is already calico 3.15.1, next 12.x releases for other providers will be as well

Reply:
Good call. I changed this to allow 3.15.